modules/engine-prototype/build.sbt

```diff
@@ -0,0 +1,24 @@
+import Dependencies._
+
+lazy val root = (project in file(".")).
+  settings(
+    inThisBuild(List(
+      organization := "edu.gemini",
+      scalaVersion := "2.12.1",
+      scalacOptions ++= Seq(
+        "-feature",
+        "-language:higherKinds",
+        "-language:existentials"
+      ),
+      version      := "0.1.0-SNAPSHOT"
+    )),
+    name := "engine",
+    libraryDependencies ++= Seq(
+      cats,
+      catsEffect,
+      fs2,
+      monocle,
+      monocleMacro,
+      scalaTest % Test
+    )
+  )
```

Comment:
This was a personal exercise in order to learn `sbt`. I used the default template to create a project. I'm not proposing anything regarding the build.